setup.py

```diff
@@ -7,7 +7,7 @@
 import sys
 from setuptools import setup, find_packages
 
-LIBRARY_VERSION = '2.0.1'
+LIBRARY_VERSION = '2.0.3'
```

Comment:
Why are you skipping `2.0.2`?

Reply:
No particular reason; let me change this back...